tensorflow/tools/ci_build/release/common.sh

```diff
@@ -140,8 +140,8 @@ function install_pip_deps {
   ${SUDO_CMD} ${PIP_CMD} install six==1.12.0
   ${SUDO_CMD} ${PIP_CMD} install grpcio
   ${SUDO_CMD} ${PIP_CMD} install portpicker
-  ${SUDO_CMD} ${PIP_CMD} install scipy
-  ${SUDO_CMD} ${PIP_CMD} install scikit-learn
+  ${SUDO_CMD} ${PIP_CMD} install scipy==1.2.3
+  ${SUDO_CMD} ${PIP_CMD} install scikit-learn==0.20.4
```

Comment:
@amahendrakar could you please confirm if these are the right scripts to bump scipy/scikit-learn versions for testing? Is there anywhere else I should bump it to ensure there are no issues?
I am adding tests that require these versions as a minimum. Scikit-learn is not required to build, only test, and the scipy version required to build does not need to change.

Reply:
@k-w-w if you are able to ping someone internally to verify this that would be very helpful, I don't want to keep asking you to kick off the tests blindly, also don't want to make unnecessary changes or find out that things break under a deployment that CI does not test for. Thanks!

Reply:
@gunan Do you know if this is the right approach to change the scikit versions?

Reply:
I'll add that I was able to reproduce the issue on my end, it definitely stems from having a newer version of scipy and an older version of scikit-learn. I had assumed that since the version was not pinned here, it was pulling the latest. But now I'm thinking maybe it has an older version cached or already installed in that image so it's just not bothering to update it. 

It would be nice if CI logged a `pip freeze`, that would help sort out problems like this in the future.

Reply:
Unfortunately, the CI uses remote build execution, which means we internally need to update some docker images, and push new containers. @av8ramit should be able to point you to instructions for that.

Reply:
Here are our docker containers for CPU/GPU presubmits.
[CPU](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.rbe.ubuntu16.04-manylinux2010)
[GPU](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/Dockerfile.rbe.cuda10.1-cudnn7-ubuntu16.04-manylinux2010)

They invoke [this script](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/install/install_pip_packages.sh) for installing pypi images.

Unfortunately, someone internally needs to update the containers after this change. @k-w-w I can send you a doc internally with how to update the images.

Reply:
@av8ramit I can't see the results from the last CI run anymore because I pushed a new commit, but I do remember the tests passed on the Mac/Python2 build. Is there any way to run again and get the package versions of scipy and scikit-learn that each presubmit runs with to at least verify the issue before making any changes to the images?